### ISSUE TITLE
Reports of LCV teal being 'hard on the eyes'

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#905]
**Package** lib-classifier

**Feature or Issue Description**
From Talk: https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/talk/2108/1020600

It is hard to determine what exactly this means. @beckyrother is suggesting trying #39565C for the background and dot #EFF2F5. This still meets accessibility contrast guidelines.

**Comment by eatyourgreens** [comment]
Maybe we should theme the viewer, then people who find it hard to use in light mode at least have the option of dark mode. A choice of colours would be more accessible than only one colour combination for everyone.

Should we record the theme in the classification too (if we don't already)?

**Comment by srallen** [comment]
> Should we record the theme in the classification too (if we don't already)?

Yeah, we can do that. 

I'm a little hesitant with adding more themes or spending too much time on this without understanding better what it is that is the issue... but I'm not sure how to go about figuring that out.

**Comment by eatyourgreens** [comment]
I think a light theme and a dark theme is fine for accessibility at the moment. I'm suggesting the viewer should be a different colour in dark mode, which seems like the smallest amount of work to me.

**Comment by srallen** [comment]
Oh, yes, I misunderstood the suggestion. @beckyrother do you have any thoughts on a color variant for dark mode?

**Comment by beckyrother** [comment]
The colors here were chosen to fit with both the dark and the light theme, but it would make sense to change based on theme. In which direction were we thinking – an even darker theme, or perhaps keep this one as the dark theme and a dark-dots-on-light for the light theme?

**Comment by eatyourgreens** [comment]
Good point. I was thinking change the dark theme, but noone says on that Talk thread whether they'd prefer a change to the light or dark theme.

**Comment by rogerhutchings** [comment]
Is there anything to do on this issue, or can it be closed?

**Comment by beckyrother** [comment]
Do we want to reverse it for the light theme and see how that feels?

**Comment by rogerhutchings** [comment]
I'm going to close this, it hasn't come up again.
